src/tiled/changeimagelayerproperties.cpp

```diff
@@ -33,7 +33,9 @@ ChangeImageLayerProperties::ChangeImageLayerProperties(
         MapDocument *mapDocument,
         ImageLayer *imageLayer,
         const QColor &color,
-        const QUrl &newSource)
+        const QUrl &newSource,
+        bool newRepeatH,
+        bool newRepeatV)
```

Comment:
That's a bit too many constructor parameters, which is starts to become annoying to use. I think the `ChangeImageLayerProperties` class could be changed to use the pattern used by `ChangeMapProperty`.

Reply:
Done.